protoc-gen-twirp/generator.go

```diff
@@ -47,6 +47,9 @@ type twirp struct {
 	importPrefix string            // String to prefix to imported package file names.
 	importMap    map[string]string // Mapping from .proto file name to import path.
 
+	// Package output:
+	paths string // instruction on where to write output files
```

Comment:
Let's make this `sourceRelativePaths bool` instead of a string with a particular value.

Reply:
Done, makes sense.  Not sure what I was thinking on that initial commit.